_data/accessibility-tests/banner.yml

```diff
@@ -0,0 +1,112 @@
+    summary_additional: |
+      When you zoom in to 400%,
+      banner content adjusts to fit the width of the screen without losing functionality.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Banner can be accessed, opened, and closed with only a keyboard.
+    summary_additional: |
+      When you navigate your site with a keyboard,
+      you can access the banner by using `tab` to move forward or `shift` + `tab` to move backwards.
+      You can use the `enter` and `spacebar` keys to open and close the banner.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: Banner does not trap keyboard focus.
+    summary_additional: |
+      When you navigate with a keyboard, you can easily move into and out of the banner,
+      regardless of whether the banner is open or closed.
+      When you navigate out of an open banner using your keyboard, it doesn't automatically close.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.2
+  - summary: Banner is the first interactive object after the main navigation.
+    summary_additional: |
+      When you navigate your site with a keyboard and skip navigation,
+      the "Here's how you know" banner button is the first focusable item on the page.
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.3
+  - summary: Focus indicator is clearly visible on banner.
+    summary_additional: |
+      When you use a keyboard to tab into the banner,
+      the "Here's how you know" button has a visible outline or other clear indication of focus.
+      On a mobile device, this focus indicator should surround the entire banner element.
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader announces button.
+    summary_additional: |
+      When you use a screen reader and access the banner,
+      the screen reader uses the word "button" to announce the "Here's how you know" button.
+    test_status:
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: Screen reader announces collapsed state.
+    summary_additional: |
+       When you first access the "Here's how you know" button while using a screen reader,
+       the screen reader announces that it is "collapsed."
+       When activated, the screen reader announces "expanded".
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 4.1.2
+  - summary: Screen reader does not announce decorative icons.
+    summary_additional: |
+       When you access and open the banner while using a screen reader,
+       you don't hear any information about the flag, blue government building, or green lock icon.
```

Comment:
@amyleadem, @amycole501, @alex-hull - Should we specify "green circled lock icon" at the end of this sentence? There are two lock icons, and without greater specificity we are asking people to rely solely on color to distinguish between the two.

Reply:
Yes at first I didn't realize there were two, either!

Reply:
Added "green circled lock icon" in [fd7120b](https://github.com/uswds/uswds-site/pull/2687/commits/fd7120b411d3f6a5e908bdcf3de1279fdbe50a74)